www/modules/migrate/controllers/outlook.client.controller.es6

```diff
@@ -1,46 +1,58 @@
 const DATE_FORMAT = 'yyyy-MM-dd';
 
 class OutlookMigrateController {
-    /** @ngInject */
-    constructor(OutlookService, $cookies, $filter, MigrateService) {
-        Object.assign(this, {
-            OutlookService,
-            MigrateService,
-            $cookies,
+  /** @ngInject */
+  constructor(OutlookService, $cookies, $filter, $stateParams, MigrateService) {
+    Object.assign(this, {
+      OutlookService,
+      MigrateService,
+      $cookies,
+      dateFormat:     DATE_FORMAT,
+      authorized:     false,
+      getEventsError: true,
+      events:         [],
+      params:         {
+        viewPath:      MigrateService.getImportClientPathView($stateParams.client),
+        startDateTime: $filter('date')(new Date(), DATE_FORMAT),
+        endDateTime:   $filter('date')(new Date(), DATE_FORMAT)
+      }
+    });
 
-            dateFormat: DATE_FORMAT,
-            authorized: false,
-            events: [],
-            params: {
-                startDateTime: $filter('date')(new Date(), DATE_FORMAT),
-                endDateTime: $filter('date')(new Date(), DATE_FORMAT)
-            }
-        });
+    this.initialize();
+  }
 
-        this.initialize();
+  initialize() {
+    if (this.$cookies['outlook_access_token'] && this.$cookies['outlook_email']) {
+      this.params.token   = this.$cookies['outlook_access_token'];
+      this.params.email   = this.$cookies['outlook_email'];
```

Comment:
Put constant string to https://docs.angularjs.org/api/auto/service/$provide#constant